spring-integration-sftp/src/test/java/org/springframework/integration/sftp/inbound/RollbackLocalFilterTests-context.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<beans xmlns="http://www.springframework.org/schema/beans"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:int="http://www.springframework.org/schema/integration"
+	xmlns:int-sftp="http://www.springframework.org/schema/integration/sftp"
+	xsi:schemaLocation="http://www.springframework.org/schema/beans http://www.springframework.org/schema/beans/spring-beans.xsd
+		http://www.springframework.org/schema/integration http://www.springframework.org/schema/integration/spring-integration.xsd
+		http://www.springframework.org/schema/integration/sftp http://www.springframework.org/schema/integration/sftp/spring-integration-sftp.xsd">
+
+	<int-sftp:inbound-channel-adapter id="sftpAdapterAutoCreate"
+			session-factory="sftpSessionFactory"
+			channel="requestChannel"
+			remote-directory-expression="'/sftpSource'"
+			local-directory="file:local-test-dir/rollback"
+			auto-create-local-directory="true"
+			filename-pattern="sftpSource1.txt"
+			local-filter="acceptOnceFilter">
+		<int:poller fixed-rate="1000" max-messages-per-poll="2" error-channel="nullChannel">
+			<int:transactional synchronization-factory="syncFactory" />
+		</int:poller>
+	</int-sftp:inbound-channel-adapter>
+
+	<int:channel id="requestChannel" />
+
+	<int:service-activator input-channel="requestChannel" ref="crash" method="handle" />
+
+	<bean id="crash" class="org.springframework.integration.sftp.inbound.RollbackLocalFilterTests$Crash" />
+
+	<bean id="acceptOnceFilter" class="org.springframework.integration.file.filters.AcceptOnceFileListFilter" />
+
+	<int:transaction-synchronization-factory id="syncFactory">
+		<int:after-rollback expression="@acceptOnceFilter.remove(payload)" />
```

Comment:
I think typically, there will only be one `ResettableFLF` in a composite filter so I think it's ok to leave it up to the user.

Also, it gets a bit complicated with the boolean result if there are more than one and not all succeed.

I'll also fix the travis problem.